pulp_container/app/models.py

```diff
@@ -503,6 +503,10 @@ class ContainerPullThroughRemote(Remote, AutoAddObjPermsMixin):
     from within a single instance of this remote.
     """
 
+    TYPE = "pull-through"
+    includes = fields.ArrayField(models.TextField(null=True), null=True)
```

Comment:
```suggestion
    TYPE = "pull-through"

    includes = fields.ArrayField(models.TextField(null=True), null=True)
```

It looks neat if we semantically separate model fields and constants.